packages/forms/src/components/RichTextInput/MenuButton.tsx

```diff
@@ -12,14 +12,13 @@ type MenuButtonProps = {
 };
 
 const MenuButton = React.forwardRef<HTMLButtonElement, MenuButtonProps>(
-  ({ active, ...rest }, ref) => (
+  ({ ...rest }, ref) => (
     <Button
       ref={ref}
-      mode="solid"
+      mode="text"
       type="button"
-      color={active ? "white" : "black"}
-      data-h2-padding="base(x.125 x.25)"
-      data-h2-font-size="base(caption)"
+      color="whiteFixed"
+      fontSize="caption"
       {...rest}
     />
```

Comment:
Was removing the active style intentional? It was being used to highlight the button to indicate the user had their cursor on that element and clicking button would toggle that back to plain text. Sort of like how other word processors work:

![Screenshot 2023-11-25 101356](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/94f4cdb8-acf3-4ba3-9107-32ed07d09cbb)
![Screenshot 2023-11-25 101402](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/2201caab-ae7a-4354-aede-9e51b6139238)


This is causing errors now because we also still pass it in on the `MenuBar` and it is still defined in the prop types. If removing the active style was intentional, we should stop passing it in and remove it from the typing for the props as well.

Reply:
THAT's what it does. I'll add it back in and look at the styling.